pilot/cmd/pilot-agent/app/wait.go

```diff
@@ -80,6 +81,27 @@ func checkIfReady(client *http.Client, url string) error {
 	return nil
 }
 
+func notifyErrorExit(client *http.Client) error {
+	url := "http://localhost:15020/quitquitquit?force=true"
```

Comment:
Nit: can we add a flag to enable killing? It can default on.

We also may need config for the URL; we already have it for the status port.